Dockerfile

```
@@ -9,3 +8,8 @@ RUN apk update && apk add go git gcc musl-dev ca-certificates mailcap \
         && go build -x -ldflags "-X main.githash=$(git log --pretty=format:'%h' -n 1)" -o /bin/matterbridge \
         && rm -rf /go \
         && apk del --purge git go gcc musl-dev
+
+FROM alpine:edge
+RUN apk --no-cache add ca-certificates
```

```suggestion
RUN apk --no-cache add ca-certificates mailcap
```
We need mailcap for mimetypes: see  #969

---

I was unsure if that was a build dependency or not. I'll update the branch.